apps/communications/dialer/test/marionette/keypad_test.js

```diff
@@ -114,4 +114,16 @@ marionette('Dialer > Keypad', function() {
     });
     assert.ok(true, 'cleaned the phone number view');
   });
+
+  test('Enabling and disabling the dialer call button', function() {
```

Comment:
These changes can go in ```apps/communications/dialer/test/unit/keypad_test.js``` instead. The structure will be a bit different, though.